### PR TITLE
CA-63 Add LinkedIn Ads template support

### PR DESCRIPTION
**Pull request by bucrogers**
# TL;DR

Freshpaint GTM Template support for the LinkedIn Ads destination.

# Summary
* Accepts up to 3 Conversion IDs, as do the two community gallery templates (one published by LinkedIn).
* The implementation here is to send up to 3 track calls, one for each Conversion ID - the same as the aforementioned gallery templates do (in their case, they're doing a LinkedIn tag call instead of track)
* This is a different use case than multi-config, where the config param differs (e.g., Facebook Pixel ID, GA4 Measurement ID) - for those, we use multiple dest instances, and specify those in the integrations object.

# Screenshot
image

### CODE REVIEW

**Review by noelle102 — APPROVED** [review]
Should definitely link this PR to a page about how to add new support to Ads template, if we have such page that is.

**Review by 626D6172 — COMMENTED** [review]
LGTM, is there a reason we only support three conversion IDs?

**Comment by bucrogers** [comment]
> LGTM, is there a reason we only support three conversion IDs?

because that's what the gallery templates support, which we'd be migrating from. I think they chose 3 to to have some kind of limit to avoid possible throttling.
